test/infrastructure/docker/docker/util.go

```diff
@@ -118,7 +118,7 @@ func list(visit func(string, *types.Node), filters ...string) error {
 	cmd := exec.Command("docker", args...)
 	lines, err := exec.CombinedOutputLines(cmd)
 	if err != nil {
-		return errors.Wrap(err, "failed to list nodes")
+		return fmt.Errorf("failed to list nodes: %w. Output: %s", err, lines)
```

Comment:
Where are we logging this? Also we usually use `pkg/errors` everywhere throughout this codebase, we should keep that

Reply:
This is not currently being logged due to https://github.com/kubernetes-sigs/cluster-api/issues/3821.